critiquebrainz/frontend/views/__init__.py

```diff
@@ -1,6 +1,7 @@
 import critiquebrainz.db.avg_rating as db_avg_rating
 import critiquebrainz.db.exceptions as db_exceptions
 
+
```

Comment:
This is unnecessary. :)

Reply:
sorry, ive seen a pep8 warning about "2 blank lines expected" in tests, and figured i'd just fix it